src/state/auth/types.js

```diff
@@ -0,0 +1,2 @@
+export const SIGN_IN = "SIGN_IN";
```

Comment:
Lets consider using more precise action types, which are dependent of some namespace(e.g. `AUTH/SIGN_IN`)